test/gui/shared/steps/steps.py

```diff
@@ -391,15 +389,23 @@ def step(context):
     clickTab(waitForObject(names.stack_QTabWidget), "Not Synced")
 
 
-def openPublicLinkDialog(context, resource):
+def openSharingDialog(context, resource, fileType='file'):
```

Comment:
```suggestion
def openSharingDialog(context, resource, itemType='file'):
```

fileType = folder sounds strange even strictly speaking a folder is a type of a file on the filesystem, maybe call it `itemType`?

Reply:
fixed